CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All Notable changes to `schema-org` will be documented in this file.
 
+## 6.0.0 - 2021-10-?
```

Comment:
This one needs a date...the subsequent next 2 should update when prior PRs are merged.